### ISSUE TITLE
I receive damage from the sword even though there's nothing in my hand.

### ISSUE THREAD

**Issue by F3F5** [hpfxd/PandaSpigot#185]
Basically, you hit the player first with a sword, then disconnect. The anti-relaunch plugin kills you, then you reconnect and hit with your hand, and there will be a drop as if from a sword.
Video From Player: https://workupload.com/file/Um88hd8nWMB
Video From Me: https://workupload.com/file/568X7YD5SMN

**Comment by jtJava** [comment]
The attributes just need updated properly

**Comment by andreasdc** [comment]
Do you have saving players enabled?
Do you have this patch? https://github.com/hpfxd/PandaSpigot/pull/153

**Comment by F3F5** [comment]
> Do you have saving players enabled? Do you have this patch? #153

Yes Enabled. And I have https://nightly.link/hpfxd/PandaSpigot/workflows/build/master/Server%20JAR.zip this file

**Comment by F3F5** [comment]
> Do you have saving players enabled? Do you have this patch? #153

tested. not working https://github.com/F3F5/PandaSpigot/commit/d7a19f984a8361719693e7a8800d16c92973ad8a

**Comment by F3F5** [comment]
@uRyanxD

**Comment by sprytex** [comment]
a

**Comment by anzz1** [comment]
Need to port this patch: https://github.com/ProjectKig/KigPaper/commit/e25a1dd8847e5c8c8b170439c1eba89a8e0492c9
the bug is MC-28289
